Jointify/Model/PoseMachineLearning/PoseNet.swift

```diff
@@ -59,12 +59,17 @@ class PoseNet {
         static let jointColor: UIColor = UIColor.systemPink
         // Degrees that will be subtraced from the measured angle
         static let neutralNullAngle: Float = 90.0
+        // Minimum confidence that each joint has to conform to
+        static let confidenceThreshold: Double = 0.70
     }
     
```

Comment:
Add `MARK` to conform to style guide
```suggestion
    
    // MARK: Stored Instance Properties
```